src/Bridges/ApplicationLatte/TemplateFactory.php

```diff
@@ -112,4 +119,10 @@ public function createTemplate(UI\Control $control = NULL)
 		return $template;
 	}
 
+	public function changeTemplateClass($templateClass)
+	{
+		\Nette\Utils\Validators::assert($templateClass, 'string');
```

Comment:
> How would you prevent passing something different than string? 

Here is the secret – you don't prevent passing something different because this is PHP because it would kill code readability and performance. Pretty much every Nette method without typehint can be broken by passing the wrong type.